src/runners/modal-runner.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WTEGModal from '../../components/modal/wtegModel/wtegModal.react';
+{
+  const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent),
```

Comment:
Could you replace this user agent sniffing with a screen size test - it will be more reilable.